lab/lab2/js/part1-ajax-calls.js

```diff
@@ -30,9 +36,9 @@ console.log("isLengthOfFiveOrMore success:",
   function you write along with underscore's _.each to log the double of every
   number in the provided array.
 ===================== */
-var logDouble = function(num) {};
+var logDouble = function(num) {console.log(num*2)};
 var theArray = [1, 5, 20, 100];
-
+_.each(theArray,logDouble);
 
 /* =====================
```

Comment:
You did not add the dataset to the map with the ajax call for the last part of this lab